scripts/probe_xas_bbh_phase.py

```
@@ -0,0 +1,87 @@
+    print(f"Ripple at 20Hz:  {ripple_phase[0]:.6f}")
+    print(f"Ripple at 100Hz: {ripple_phase[np.argmin(np.abs(fb-100))]:.6f}")
+
+    # Subtract value at fRef first (proxy for "remove phifRef constant")
+    iref = np.argmin(np.abs(fb - f_ref))
+    ripple_norm = ripple_phase - ripple_phase[iref]
+    lal_norm = lal_phi - lal_phi[iref]
+    diff = ripple_norm - lal_norm
+    A = np.vstack([fb - f_ref, np.ones_like(fb)]).T
+    coef, *_ = np.linalg.lstsq(A, diff, rcond=None)
+    resid = diff - A @ coef
+    print(f"\n[After subtracting value at fRef]")
```

Ruff (0.15.13)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)